irc/hostserv.go

```diff
@@ -316,3 +366,60 @@ func hsRejectHandler(server *Server, client *Client, command string, params []st
 		}
 	}
 }
+
+func hsForbidHandler(server *Server, client *Client, command string, params []string, rb *ResponseBuffer) {
+	user := params[0]
+	forbidden := command == "forbid"
+
+	_, err := server.accounts.VHostForbid(user, forbidden)
+	if err == errAccountDoesNotExist {
+		hsNotice(rb, client.t("No such account"))
+	} else if err != nil {
+		hsNotice(rb, client.t("An error occurred"))
+	} else {
+		if forbidden {
+			hsNotice(rb, fmt.Sprintf(client.t("Successfully forbidden vhosts to user %s"), user))
+		} else {
+			hsNotice(rb, fmt.Sprintf(client.t("Successfully permitted vhosts for user %s"), user))
+		}
+	}
+}
+
+func hsOfferListHandler(server *Server, client *Client, command string, params []string, rb *ResponseBuffer) {
+	vhostConfig := server.Config().Accounts.VHosts
+	if len(vhostConfig.OfferList) == 0 {
+		if vhostConfig.UserRequests.Enabled {
+			hsNotice(rb, client.t("The server does not offer any vhosts (but you can request one with /HOSTSERV REQUEST)"))
```

Comment:
instead of the `()`, we can just use a comma here. easier to read and all:  "The server does not offer any vhosts, but you can request one with /HOSTSERV REQUEST"